### DISCUSSION TITLE
Create ERD before database

### DISCUSSION THREAD

**Discussion by pixel24** [dbeaver/dbeaver#19978] [General]
Hi@all,

I am new to both SQL and DBeaver. I need this for training.

I use the community version (version 23.0.4.202305071722) on Linux. When I create my databases and tables DBeaver shows me at the end the relationships between the tables. This is very helpful.

However, before I create a database for real I would like to design the ERD. Is this possible in DBeaver?

Or do I just not find it :-(

with best
pixel24

**Comment by pixel24** [comment]
Ok, this seems to work only with the Enterprise version :-(

**Reply by LonwoLonwo** [reply]
Yes, it is true. It works only in our [PRO products](https://dbeaver.com/edition/).

**Comment by pixel24** [comment]
I'm currently in a retraining course to become an IT specialist because
you're rather tight on cash. So I will work with the community version.

Thank you for the feedback.

Am Mo., 22. Mai 2023 um 12:21 Uhr schrieb Anastasiya <
***@***.***>:

**Comment by joseluisbz** [comment]
Hi there...
This feature is not available in Pro versions according to https://x.com/joseluisbz/status/1769814878632505646?s=20

**This is my Question:**
_[@dbeaver_news](https://twitter.com/dbeaver_news)
 Hi teams! Would you mind telling me if is possible to create my total ERD first only, and when the ERD is finished then select one Database Engine, (Oracle, SQL Server, PostgreSQL or MySQL) to create my Database.
Thanks in advance._

**Their answer was:**
_Thank you for the question. Unfortunately, DBeaver has no such capability._

**I told them:**
_Thank you for answer.
I found something related:
[https://dbdesigner.net](https://t.co/tHZbYBhCEj)
and
[https://dbdiagram.io](https://t.co/9emrG7wl6L)
But, I think that feature would be valuable in order to design first and create DB later._

**And they answered to me:**
_Thank you for sharing. If you don't mind, please create a feature request on our GitHub:
https://github.com/dbeaver/dbeaver/
If our team gets more requests for such a functionality, it would be easier to decide to implement it_

**### Then, I need to ask for this feature!**
